src/screens/ImportWalletScreen.tsx

```diff
@@ -32,12 +33,14 @@ interface Props {
       StackNavigationProp<MainCardStackNavigatorParams, Route.ImportWallet>
     >
   >;
+  route: RouteProp<MainCardStackNavigatorParams, Route.ImportWallet>;
   loadWallets: () => Promise<WalletsActionType>;
 }
 
 export const ImportWalletScreen = (props: Props) => {
```

Comment:
this component has got so many of its own methods that it shouldn't be a functional component as it creates functions at every rerender. Please make it standard Component